lib/modules/fuzzer/index.js

```diff
@@ -17,14 +17,18 @@ class ContractFuzzer {
   generateFuzz(iterations, contract) {
     const self = this;
     let fuzzMap = {};
-    for (let i = 0; i < iterations; i++) {
-      contract.abiDefinition.filter((x) => x.inputs && x.inputs.length != 0).forEach((abiMethod) => {
-        let name = abiMethod.type === "constructor" ? "constructor" : abiMethod.name;
-        let inputTypes = abiMethod.inputs.map(input => input.type);
-        fuzzMap[name] = inputTypes.map(input => this.getTypeFuzz(input));
-      });
-    }
-    for (let key in fuzzMap) self.logger.info(key + ":" + fuzzMap[key]);
+    contract.abiDefinition.filter((x) => x.inputs && x.inputs.length != 0).forEach((abiMethod) => {
+      let name = abiMethod.type === "constructor" ? "constructor" : abiMethod.name;
+      let inputTypes = abiMethod.inputs.map(input => input.type);
+      fuzzMap[name] = {};
+      for (let i = 0; i < iterations; i++) {
+        fuzzMap[name][i] = {};
```

Comment:
This line is useless as it is replaced by what's on the next line

Reply:
you need to define it otherwise it gives me an undefined error?

Reply:
The one on line 23 is necessary, that one no